src/nativestore/NodeBlock.h

```diff
@@ -25,18 +25,20 @@ class RelationBlock;  // Forward declaration
 #define NODE_BLOCK
 
 class NodeBlock {
- private:
+   private:
     bool isDirected = false;
 
- public:
+   public:
     unsigned int addr = 0;
     std::string id = "";  // Node ID for this block ie: citation paper ID, Facebook accout ID, Twitter account ID etc
     char usage = false;   // Whether this block is in use or not
-    unsigned int edgeRef = 0;      // edges database block address for relations
+    unsigned int nodeId ;      // nodeId for each block
+    unsigned int edgeRef = 0;      // edges database block address for relations size of edgeRef is 4 bytes
+    unsigned int centralEdgeRef = 0; // edges cut database block address for edge cut relations
     unsigned char edgeRefPID = 0;  // Partition ID of the edge reference
     unsigned int propRef = 0;      // Properties DB block address for node properties
 
-    static const unsigned long BLOCK_SIZE = 16;  // Size of a node block in bytes
+    static const unsigned long BLOCK_SIZE = 24;  // Size of a node block in bytes
```

Comment:
Why is the `BLOCK_SIZE` changed from 16 to 24?

Reply:
In each Node Block we store information with size of 24 Bytes. Please refer to lines 46-54

Reply:
Ok